includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -1986,9 +1985,13 @@ private function finalize_styles() {
 			}
 
 			$include_manifest_comment = (
-				'always' === $this->args['include_manifest_comment']
-				||
-				( $excluded_size > 0 && 'when_css_excluded' === $this->args['include_manifest_comment'] )
+				'never' !== $this->args['include_manifest_comment']
```

Comment:
@westonruter Why did you add this? The value of `include_manifest_comment` is already tested in the second condition. This won't change the logic.